lib/matplotlib/backend_bases.py

```diff
@@ -3596,29 +3592,62 @@ def _message_cbk(self, event):
         """Captures the 'tool_message_event' to set the message on the toolbar"""
         self.set_message(event.message)
 
-    def _tool_triggered_cbk(self, event):
+    def _tool_toggled_cbk(self, event):
         """Captures the 'tool-trigger-toolname
 
         This only gets used for toggled tools
         """
-        if event.sender is self:
```

Comment:
Correct. I don't see any reason to outright ban it, i.e. putting in safeguards to prevent it.  So we might as well go the whole hog and allow it.  It doesn't use up any extra lines of code, and we gain an extra feature, compared to banning it which would add more lines of code.